src/components/Text/__stories__/Text.stories.mdx

```diff
@@ -138,7 +138,7 @@ When the text is longer than its container and the ellipsis flag is on, the end
 We support two kinds of ellipsis: single-line ellipsis with a tooltip displayed in hover or ellipsis after multiple lines. You can see examples for both use cases below.
 
 <Canvas>
-  <Story name="Overflow" play={textOverflowSuite}>
+  <Story name="Overflow" play={textOverflowSuite} parameters={{ status: "new" }}>
```

Comment:
This is the #1 of passing the status, but doing `name="Overflow [new]"` would do the same